src/frontend/js/components/CourseRunEnrollment/index.tsx

```diff
@@ -259,17 +261,13 @@ export const CourseRunEnrollment: React.FC<
 
     case state.matches('anonymous'):
       return (
-        <React.Fragment>
-          <button
-            className="course-run-enrollment__cta disabled"
-            aria-disabled="true"
-          >
-            <FormattedMessage {...messages.enroll} />
-          </button>
-          <div className="course-run-enrollment__helptext">
-            <FormattedMessage {...messages.loginToEnroll} />
-          </div>
-        </React.Fragment>
+        <a
+          href={loginUrl}
+          className="course-run-enrollment__cta"
+          title={intl.formatMessage(messages.loginToEnroll)}
```

Comment:
I think there is no purpose in putting the same thing as link title as in the actual link text (instead we ca just omit the title).

As it is I think it might be flagged as a non-conformity in an accessibility audit

Reply:
Si I should set a different title or remove it ?

Reply:
IMO remove it